nixos/modules/programs/firefox.nix

```diff
@@ -204,46 +246,35 @@ in
       '';
     };
 
-    nativeMessagingHosts = mapAttrs (_: v: mkEnableOption (mdDoc v)) {
-      browserpass = "Browserpass support";
-      bukubrow = "Bukubrow support";
-      euwebid = "Web eID support";
-      ff2mpv = "ff2mpv support";
-      fxCast = "fx_cast support";
-      gsconnect = "GSConnect support";
-      jabref = "JabRef support";
-      passff = "PassFF support";
-      tridactyl = "Tridactyl support";
-      ugetIntegrator = "Uget Integrator support";
-    };
+    nativeMessagingHosts = ({
+      packages = mkOption {
```

Comment:
Stupid question: what else could we put here that justifies adding packages? Otherwise we could just name it ``nativeMessagingHostPackages`` and/or rename it later when it is actually required.

Reply:
All the existing options are already under there, so we can't get rid of the namespace (yet).

Reply:
I'd like to eventually just get rid of all the options and make `nativeMessagingHosts` a list.

Reply:
We'll have a hard break when the deprecated options are removed.

Reply:
until then users are informed with a warning